internal/httpclient/manager_test.go

```
@@ -739,3 +742,152 @@ func BenchmarkFingerprintGeneration(b *testing.B) {
+	const (
+		rounds  = 60
+		hitters = 3
+	)
+
+	for i := 0; i < rounds; i++ {
+		manager := NewHTTPClientManager()
+		manager.maxClients = 2
+
+		// Distinct timeouts per round keep fingerprints unique across rounds.
+		base := time.Duration(i * 10)
+		configA := &Config{RequestTimeout: (1 + base) * time.Second}
```

golangci-lint (2.12.2)

[error] Multiplication of durations: `(1 + base) * time.Second`

(durationcheck)